hw/dv/tools/dvsim/questa.hjson

```diff
@@ -0,0 +1,106 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  build_cmd:  "vlib work && {job_prefix} vlog"
+  run_cmd:    "{job_prefix} vsim"
+
+  // Indicate the tool specific helper sources - these are copied over to the
+  // {tool_srcs_dir} before running the simulation.
+  tool_srcs:  ["{dv_root}/tools/questa/*"]
```

Comment:
Note that `tool_srcs` has been deprecated in #4538. These will need to be removed.

Reply:
I removed this line 10 in new PR